### ISSUE TITLE
Imageoverlaypalette for file cannot be deactivated

### ISSUE THREAD

**Issue by evelineketterer** [Gernott/mask#619]
If I add an image to the mask element and deactivate imageoverlaypalette, the fields still appear in the content element: alternative, link, crop

T312, php 8.1

**Comment by nhovratov** [comment]
@evelineketterer yes, this is known to me. The reason is a change in TYPO3 v12 to enable it by default. The Mask checkbox to enable it has no effect anymore. Will be removed in the next major version.

**Comment by nhovratov** [comment]
If you still want to disable it, you will need to override TCA right now.
